pkg/local_workflows/data_transformation_workflow.go

```diff
@@ -61,43 +74,43 @@ func dataTransformationEntryPoint(invocationCtx workflow.InvocationContext, inpu
 	}
 	if sarifInput == nil || summaryInput == nil {
 		logger.Trace().Msg("incomplete input data for transformation")
-		return output, nil
+		return input, nil
```

Comment:
Suggestion: seems like we didn't cover any of these cases in the tests, sounds reasonable to extend the tests. WDT?

Reply:
Added a test case for incomplete input 👍